canonical-web-and-design/ubuntu.com
# Stop automatically redirecting Chinese language users

## Done

- Add a notice on the homepage to inform Chinese users of the cn.ubuntu.com website rather than redirect them, just like the Japanese language one.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Change your browser's language to Chinese and see the notice on the homepage

## Issue / Card

Fixes #6146

## Screenshots

![image](https://user-images.githubusercontent.com/441217/69097536-a7f80e80-0a4e-11ea-87b4-922ff7692872.png)
